app/src/main/java/com/framgia/wsm/screen/confirmrequestoff/ConfirmRequestOffViewModel.java

```diff
@@ -243,9 +243,53 @@ public String getTitleToolbar() {
         return mContext.getString(R.string.confirm_edit_request_off);
     }
 
+    public String getStartDateHaveSalary() {
+        if (mOffRequest != null && mOffRequest.getStartDayHaveSalary() != null) {
+            if (!mOffRequest.getStartDayHaveSalary().getOffPaidFrom().equals("")) {
+                return mOffRequest.getStartDayHaveSalary().getOffPaidFrom()
+                        + Constant.BLANK
+                        + mOffRequest.getStartDayHaveSalary().getPaidFromPeriod();
+            }
+        }
+        return "";
+    }
+
+    public String getEndDateHaveSalary() {
+        if (mOffRequest != null && mOffRequest.getEndDayHaveSalary() != null) {
+            if (!mOffRequest.getEndDayHaveSalary().getOffPaidTo().equals("")) {
+                return mOffRequest.getEndDayHaveSalary().getOffPaidTo()
+                        + Constant.BLANK
+                        + mOffRequest.getEndDayHaveSalary().getPaidToPeriod();
+            }
+        }
+        return "";
+    }
+
+    public String getStartDateNoSalary() {
+        if (mOffRequest != null && mOffRequest.getStartDayNoSalary() != null) {
+            if (!mOffRequest.getStartDayNoSalary().getOffFrom().equals("")) {
+                return mOffRequest.getStartDayNoSalary().getOffFrom() + Constant.BLANK + mOffRequest
+                        .getStartDayNoSalary()
+                        .getOffFromPeriod();
+            }
+        }
+        return "";
+    }
+
+    public String getEndDateNoSalary() {
+        if (mOffRequest != null && mOffRequest.getEndDayNoSalary() != null) {
+            if (!mOffRequest.getEndDayNoSalary().getOffTo().equals("")) {
```

Comment:
WARNING: [CollapsibleIfStatements] These nested if statements could be combined
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/basic.html#CollapsibleIfStatements